fe/fe-core/src/main/java/com/starrocks/catalog/StructType.java

```diff
@@ -180,6 +180,27 @@ public StructField getField(int pos) {
         return fields.get(pos);
     }
 
+    public void updateFields(List<StructField> structFields) {
+        Preconditions.checkNotNull(structFields);
+        Preconditions.checkArgument(structFields.size() > 0);
+        fields.clear();
+        fieldMap.clear();
+        for (StructField field : structFields) {
+            LOG.info("field name " + field.getName());
```

Comment:
better to print a final log with all struct fields, instead of a log per field.

Reply:
> better to print a final log with all struct fields, instead of a log per field.

done.